helix-core.el

```diff
@@ -148,12 +148,18 @@ If a region is already active, no new region is created."
        (unless (use-region-p)
          (push-mark current t 'activate)))))
 
-(defun helix-forward-word ()
-  "Move to next word."
+(defun helix-forward-word-start ()
```

Comment:
Since these are public, I think this is worthy of `define-obsolete-function-alias` so we don't break consumers:

```elisp
(define-obsolete-function-alias 'helix-forward-word 'helix-forward-word-start "0.8.0")
```

ditto for `helix-forward-long-word`